.env.example

```
@@ -26,20 +26,23 @@ BOT_GITHUB_TOKEN=1a2b3c4d5e6f7g8g9i0j
 # Feature: basically everything
 # Mocked: No, must run sqlite locally
 DATABASE_FILENAME="sqlite.db"
+DATABASE_PATH="./prisma/sqlite.db"
 DATABASE_URL="file:./prisma/sqlite.db"
 CACHE_DATABASE_PATH="other/cache.db"
 LITEFS_DIR="./prisma"
+PORT=3000
 
 # Feature: Fly
 # Mocked: No
+FLY_APP_NAME="kcd"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)